### PR TITLE
Add Netlify toml as alternative set up.

### PR DESCRIPTION
**Pull request by hkd987**
Allow users to copy paste toml file settings into project

### CODE REVIEW

File: docs/deploy-to-netlify.md [hunk 1]
```diff
@@ -8,4 +8,14 @@ To deploy your Gridsome site to Netlify, go to the create a new site page, selec
 
 Add these build settings:
 - **Build Command:** `gridsome build`
-- **Publish directory:** `dist`
\ No newline at end of file
+- **Publish directory:** `dist`
+
+Alternative deploy using toml file:
```
Comment (12vanblart) [inline docs/deploy-to-netlify.md]: Quick Grammar fix: 

"Alternatively, deploy using a `.toml` file"

File: docs/deploy-to-netlify.md [hunk 2]
```diff
@@ -8,4 +8,14 @@ To deploy your Gridsome site to Netlify, go to the create a new site page, selec
...
+  ```
+  [build]
+    publish = "dist"
+    command = "gridsome build"
+  ```
+ 
+ Create a file in the root of your project called `netlify.toml` this file should have the above configuation.
```
Comment (12vanblart) [inline docs/deploy-to-netlify.md]: Thoughts on adding a "For more configuration via `.toml` check here" with a link to https://www.netlify.com/docs/netlify-toml-reference/ ?

Comment (hkd987) [reply docs/deploy-to-netlify.md]: I've made this change

**Comment by hkd987** [comment]
@12vanblart Ive made all changes request.

As a side note i'll be working on more changes to the docs in the next couple days // weeks. I just completed a large multi connected project using Netlify CMS and have some thoughts.

**Review by thetre97 — APPROVED** [review]

**Pull request merged**